### ISSUE TITLE
[BUG] Create HANA Cloud instance (hana) failed

### ISSUE THREAD

**Issue by michael1011101** [SAP/terraform-provider-btp#520]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What version of Terraform are you using?

0.6.0-beta1

### What type of issue are you facing

bug report

### Describe the bug

After executing `terraform apply`, the terraform plugin crashed and sent the following error msg.

But on the btp cockipt, the instance was still creating.

```
Plan: 2 to add, 0 to change, 0 to destroy.
btp_subaccount_service_instance.hana_cloud_hana_instance: Creating...
╷
│ Error: Plugin did not respond
│ 
│ The plugin encountered an error, and failed to respond to the plugin6.(*GRPCProvider).ApplyResourceChange call. The plugin logs may
│ contain more details.
╵

Stack trace from the terraform-provider-btp_v0.6.0-beta1 plugin:

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x28 pc=0x1728caf]

goroutine 43 [running]:
github.com/SAP/terraform-provider-btp/internal/provider.subaccountServiceInstanceValueFrom({_, _}, {{0xc0002ca300, 0x24}, 0x0, 0x0, {0x0, 0x0}, {0x0, 0x0}, ...})
        github.com/SAP/terraform-provider-btp/internal/provider/type_subaccount_service_instance.go:44 +0x12f
github.com/SAP/terraform-provider-btp/internal/provider.(*subaccountServiceInstanceResource).Create(0xc000186080, {0x1a2e230?, 0xc0002c4cc0}, {{{{0x1a329d8, 0xc0005423c0}, {0x17d8220, 0xc0002c5f80}}, {0x1a351f0, 0xc0001385f0}}, {{{0x1a329d8, ...}, ...}, ...}, ...}, ...)
        github.com/SAP/terraform-provider-btp/internal/provider/resource_subaccount_service_instance.go:197 +0x848
github.com/hashicorp/terraform-plugin-framework/internal/fwserver.(*Server).CreateResource(0xc0001eb080, {0x1a2e230, 0xc0002c4cc0}, 0xc0005b3390, 0xc0005b3330)
        github.com/hashicorp/terraform-plugin-framework@v1.4.2/internal/fwserver/server_createresource.go:101 +0x578
github.com/hashicorp/terraform-plugin-framework/internal/fwserver.(*Server).ApplyResourceChange(0xc0005414e8?, {0x1a2e230, 0xc0002c4cc0}, 0xc0002c8410, 0xc0005b34e8)
        github.com/hashicorp/terraform-plugin-framework@v1.4.2/internal/fwserver/server_applyresourcechange.go:57 +0x4a5
github.com/hashicorp/terraform-plugin-framework/internal/proto6server.(*Server).ApplyResourceChange(0xc0001eb080, {0x1a2e230?, 0xc0002c4ba0?}, 0xc0002c8370)
        github.com/hashicorp/terraform-plugin-framework@v1.4.2/internal/proto6server/server_applyresourcechange.go:55 +0x3ff
github.com/hashicorp/terraform-plugin-go/tfprotov6/tf6server.(*server).ApplyResourceChange(0xc0002766e0, {0x1a2e230?, 0xc0002c41b0?}, 0xc00026a150)
        github.com/hashicorp/terraform-plugin-go@v0.19.0/tfprotov6/tf6server/server.go:857 +0x56a
github.com/hashicorp/terraform-plugin-go/tfprotov6/internal/tfplugin6._Provider_ApplyResourceChange_Handler({0x18ae440?, 0xc0002766e0}, {0x1a2e230, 0xc0002c41b0}, 0xc00026a0e0, 0x0)
        github.com/hashicorp/terraform-plugin-go@v0.19.0/tfprotov6/internal/tfplugin6/tfplugin6_grpc.pb.go:467 +0x169
google.golang.org/grpc.(*Server).processUnaryRPC(0xc0002681e0, {0x1a33a00, 0xc000102ea0}, 0xc000506000, 0xc0003ce090, 0x1f3de38, 0x0)
        google.golang.org/grpc@v1.57.1/server.go:1358 +0xe15
google.golang.org/grpc.(*Server).handleStream(0xc0002681e0, {0x1a33a00, 0xc000102ea0}, 0xc000506000, 0x0)
        google.golang.org/grpc@v1.57.1/server.go:1735 +0x9e7
google.golang.org/grpc.(*Server).serveStreams.func1.1()
        google.golang.org/grpc@v1.57.1/server.go:970 +0xbb
created by google.golang.org/grpc.(*Server).serveStreams.func1 in goroutine 24
        google.golang.org/grpc@v1.57.1/server.go:981 +0x145

Error: The terraform-provider-btp_v0.6.0-beta1 plugin crashed!

This is always indicative of a bug within the plugin. It would be immensely
helpful if you could report the crash with the plugin's maintainers so that it
can be fixed. The output above should help diagnose the issue.
```

### Expected Behavior

Create HANA Cloud instance successfully.

### Steps To Reproduce

```
resource "btp_subaccount_service_instance" "hana_cloud_hana_instance" {
  subaccount_id = data.btp_subaccount.my_account.id
  # The service plan ID can be looked up via the data source btp_subaccount_service_plan
  serviceplan_id = data.btp_subaccount_service_plan.by_name.id
  name           = var.service_instance_name
  parameters     = jsonencode({
    "data" = {
      "memory" = 30,
      "systemPassword" = "xxPasswordxx",
      "edition" = "cloud"
    }
  })
  
  # var.service_instance_parameters
  timeouts = {
    create = "25m"
    update = "15m"
    delete = "15m"
  }
}
```

**Comment by lechnerc77** [comment]
Reproducible on canary

**Comment by lechnerc77** [comment]
@michael1011101 : FIx is in preparation and will  be shipped with a new release next week. 

For the time being if this blocks your work, please downgrade to provider version 0.5.0-beta1 (be aware to use Terraform CLI 1.5.7 with this release)

**Comment by michael1011101** [comment]
I tried with terraform CLI v1.5.7 and sap/btp provider with v0.5.0-beta1, and it works now. 
Need to downgrade terraform CLI version because of [this issue -- does not find resource/data source types with TF 1.6](https://github.com/SAP/terraform-provider-btp/issues/506), right?

**Comment by lechnerc77** [comment]
> I tried with terraform CLI v1.5.7 and sap/btp provider with v0.5.0-beta1, and it works now. Need to downgrade terraform CLI version because of [this issue -- does not find resource/data source types with TF 1.6](https://github.com/SAP/terraform-provider-btp/issues/506), right?

Yes this is correct. We will push a new release today, latest tomorrow that contains the fix.

**Comment by lechnerc77** [comment]
@michael1011101 version 0.6.0-beta2 is available which contains the fix and is compatible with version 1.6.x of the Terraform CLI
